packages/cost/models/authors/deepseek/deepseek-v3/model.ts

```diff
@@ -34,6 +34,17 @@ export const models = {
     modality: { inputs: ["text"], outputs: ["text"] },
     tokenizer: "DeepSeek"
   },
+  "deepseek-v3.2-speciale": {
+    name: "DeepSeek V3.2 Speciale",
+    author: "deepseek",
+    description:
+      "DeepSeek 3.2 Speciale is an advanced AI model engineered for superior reasoning and problem-solving, particularly in technical domains like coding and mathematics. This specialized version offers greater accuracy and nuanced understanding for complex tasks while remaining versatile for general conversation.",
+    contextLength: 163_840,
```

Comment:
Can you double check this context length? Online I'm seeing the context length is `131_072`.